gui/webdisplay/src/TWebWindow.cxx

```diff
@@ -243,7 +491,7 @@ bool ROOT::Experimental::TWebWindow::ProcessWS(THttpCallArg &arg)
 
    if (arg.IsMethod("WS_READY")) {
 
-      auto conn = FindConnection(arg.GetWSId(), true);
+      auto conn = FindConnection(arg.GetWSId(), true, arg.GetQuery());
```

Comment:
What is `true`? Consider adding a separate function for `true`/`false` (not as part of this PR).

Reply:
Done anyway